src/js/main.js

```diff
@@ -1,9 +1,30 @@
-let header = document.querySelector('header');
-
-window.addEventListener("scroll",()=> {
-    if(window.scrollY > 600){
-      header.style.boxShadow= "0 1px 4px  hsl(218, 13%, 48%)";
-    } else {
-        header.style.boxShadow= "none";
-    }
-})
\ No newline at end of file
+const header = document.querySelector('header');
+
+window.addEventListener('scroll', () => {
+  if (window.scrollY > 600) {
+    header.style.boxShadow = '0 1px 4px  hsl(218, 13%, 48%)';
+  } else {
+    header.style.boxShadow = 'none';
+  }
+});
+
+const navIcon = document.querySelector('.mobile-menu');
+const mainNav = document.querySelector('.main-nav');
+
+navIcon.addEventListener('click', () => {
+  mainNav.style.height = '100%';
+  mainNav.classList.toggle('open');
+  if (mainNav.classList.contains('open')) {
+    navIcon.src = './src/images/close-icon.svg';
```

Comment:
- [ ] This icon does not load immediately when clicked on the mobile menu options. I noticed that I needed to always refresh the page for it to load again. I think this is not normal. When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears and not the hamburger button. Kindly fix this issue.

Reply:
Thank you fro your feedback, I have fixed the mentioned issue